src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -1,19 +1,208 @@
 <template>
-    <p>Run workflow progress is coming soon</p>
+    <div
+        class="container mt-3"
+        id="workflow-progress-container"
+        v-if="workflowRunSummaries"
+    >
+        <div class="row mb-3">
+            <label for="workflows" class="form-label col">Workflow</label>
+            <div class="col-10 px-0">
+                <v-select
+                    :options="workflowRunSummaries.data"
+                    label="name"
+                    :reduce="(label) => label.key"
+                    name="workflows"
+                    id="v-select"
+                    v-model="selectedWorkflowKey"
+                    placeholder="Select workflow or search by name..."
+                >
+                    <template #option="{ name, date }">
+                        <div>
+                            {{ name }}
+                            <span style="opacity: 0.5; float: right">
+                                {{ formatDate(date) }}</span
+                            >
+                        </div>
+                    </template>
+                </v-select>
+            </div>
+        </div>
+        <div class="row" v-if="workflowRunStatus" id="workflow-table">
```

Comment:
The implementation plan states that this information should be updated automatically by polling the server, as we do for report progress. This should be a reasonably simple addition via a `setInterval()`. Otherwise there's no way to get updates - there's no refresh button and reloading the page deselects the relevant workflow.

Reply:
Maybe that could be spun out into a separate ticket, it would be really good to get this merged.

Reply:
Fine me with: @Nicolas-Dolan please create a ticket if you agree

Reply:
mrc-2516